avocado/utils/stacktrace.py

```diff
@@ -27,7 +27,7 @@ def prepare_exc_info(exc_info):
     return "".join(tb_info(exc_info))
 
 
-def log_exc_info(exc_info, logger='root'):
+def log_exc_info(exc_info, logger=''):
```

Comment:
True, but it makes sense to also change the docstring: "defaults to an empty string, which selects the root logger"